docs/authentication/embedded-sdk-add-passkeys.mdx

```diff
@@ -58,109 +58,117 @@ If your application is using the [NextAuth](https://next-auth.js.org) provider (
 
-  :::note PKCE
-  Note that the following query parameters includes [PKCE](https://www.rfc-editor.org/rfc/rfc7636) as it is recommeded, but optional. If you send an authorization request with PKCE, you must store the hash of the `code_challenge` so that it can be passed to the token exchange endpoint later as a `code_verifier`.
-  :::
+:::note PKCE
+Note that the following query parameters includes [PKCE](https://www.rfc-editor.org/rfc/rfc7636) as it is recommeded, but optional. If you send an authorization request with PKCE, you must store the hash of the `code_challenge` so that it can be passed to the token exchange endpoint later as a `code_verifier`.
+:::
 
-1. Generate a random string from your application for **STATE** value and is returned back to you in the response. 
+1. Generate a random string from your application for **STATE** value and is returned back to you in the response.
 
-  The `STATE` parameter mitigates [CSRF attacks](https://en.wikipedia.org/wiki/Cross-site_request_forgery) by embedding a unique value for each authentication.  
+The `STATE` parameter mitigates [CSRF attacks](https://en.wikipedia.org/wiki/Cross-site_request_forgery) by embedding a unique value for each authentication.
 
-1. Click the **Authenticator Config** tab and set the Invoke URL to a URL that _points_ to your application. <mark>Isn't this done when you create the app because the Invoke URL field is required? Could we say "...make sure the Invoke URL points to your application"?</mark> 
+1. Click the **Authenticator Config** tab and set the Invoke URL to a URL that _points_ to your application. <mark>Isn't this done when you create the app because the Invoke URL field is required? Could we say "...make sure the Invoke URL points to your application"?</mark>
 
-  In the case of a native application (iOS, Android, Flutter, React Native), this is either an App Scheme or a Universal URL / App Link. In the case of a web application, this is just a URL to your web application or a specific page of your web application.
+In the case of a native application (iOS, Android, Flutter, React Native), this is either an App Scheme or a Universal URL / App Link. In the case of a web application, this is just a URL to your web application or a specific page of your web application.
 
-  :::caution
-  While app schemes are generally easier to set up, Universal URLs and App Links are recommended as they provide protection against App Scheme hijacking.
-  :::
+:::caution
+While app schemes are generally easier to set up, Universal URLs and App Links are recommended as they provide protection against App Scheme hijacking.
+:::
 
 1. Start authorization for Invocation Type.
 
-  <Tabs groupId="authenticate-invocation-type" queryString>
+<Tabs groupId="authenticate-invocation-type" queryString>
```

Comment:
I think some of these spaces might have been intentional to nest under the number 1.  for example. At least that’s a lot of the changes I just made in my nextjs PR that just merged.

Reply:
Patricia already merged this, but I think she's adding back in some of the appropriate indentations.